wazuh/wazuh-kibana-app
# 4711 error not caught and wrong error message in pci module

### Description

Handling endpoint response was done when there is no data to show
 
### Issues Resolved

[#4711 ]

### Evidence

### Before

![image](https://user-images.githubusercontent.com/99441266/203619880-690106ab-fc3c-4f0e-b948-49a922bf752a.png)

### After
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/99441266/203616640-d698c48e-ab25-4150-95b0-dc85660ba2e5.png">

### Steps to test

*Click on the PCI module
*Click on 'Controls' tab
*Verify that no error message is displayed in UI
